MediaManager/Plugin.MediaManager.Android/MediaPlayerService.cs

```diff
@@ -353,11 +355,14 @@ private set
         /// <summary>
         /// Intializes the player.
         /// </summary>
-        public async Task Play(string url)
+        public async Task Play(string url, MediaFileType type)
```

Comment:
I think I don't understand you completely. It's not a change in the public interface.
How else should we pass in the MediaFileType?